provider/haproxy/config/haproxy_template.cfg

```diff
@@ -82,6 +82,6 @@ mode tcp
 {{else -}}
 mode {{$backend.Protocol}}
 {{end -}}
-{{range $j, $ep := $backend.Endpoints}}server {{$ep.Name}} {{$ep.IP}}:{{$ep.Port}} {{$ep.Config}}
+{{range $j, $ep := $backend.Endpoints}}server {{$ep.Name}} {{$ep.IP}}:{{$ep.Port}} {{ if $ep.Config }}{{$ep.Config}}{{else}} cookie {{ $j }}{{end}}
```

Comment:
looks like it makes custom config and cookie directive become mutually exclusive? Also we already append "cookie <cookie value> to the server here:

https://github.com/ibrokethecloud/lb-controller/blob/master/provider/haproxy/haproxy_config_util.go#L227 (if the cookie policy is specified)

Another thing is - cookie value should be consistent, and persist the same for the endpoint even when its order got changed (new ip was added to the server list, ip was removed, etc). So just the iteration id for the cookie order might not be the best solution.